### PR TITLE
Position in PLURAL

### PR DESCRIPTION
**Pull request by mwjames**
Follow-up on https://github.com/SemanticMediaWiki/SemanticMediaWiki/commit/c2e5daa762373a2c6255b3ace454d78377f0ad3c#commitcomment-17451900

### CODE REVIEW

**Comment by mwjames** [comment]
@kghbln I'm entrusting you with the merge.

File: i18n/en.json [hunk 1]
```diff
@@ -204,7 +204,7 @@
 	"smw-ask-sorting": "Sorting",
 	"smw-ask-search": "Search",
 	"smw-ask-format-selection-help": "For a detailed description, please visit the $1 help page.",
-	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code> (from cache)|<code>$2</code>}} in $4 {{PLURAL:$4|second|seconds}}.",
+	"smw-ask-query-search-info":"The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|<code>$2</code>|<code>$2</code> (from cache)}} in $4 {{PLURAL:$4|second|seconds}}.",
```
Comment (Nikerabbit) [inline i18n/en.json]: I don't see this doing what I asked. I am expecting to see `{{PLURAL:$3|1=<code>$2</code>|<code>$2</code> (from cache)}}` or something similar.

Comment (mwjames) [inline i18n/en.json]: > I don't see this doing what I asked

Not sure I understand the expectation. So you want a number as in `1=<code>$2</code> (from cache)` and `0=<code>$2</code>`?

Comment (Nikerabbit) [inline i18n/en.json]: If you have `{{PLURAL:$1|1=something|other}}` it basically means that language specific plural rules are ignored: first parameter is only used for number 1 (and not whatever the language considers singular), and the other parameter for everything else. I don't mind which part is `something` and which part `other`.

Comment (kghbln) [inline i18n/en.json]: From what I have read this should be it (assuming that true is singular = from cache):

```The query <code><nowiki>$1</nowiki></code> was answered by the {{PLURAL:$3|1=<code>$2</code> (from cache)|<code>$2</code> (from cache)|<code>$2</code>}} in $4 {{PLURAL:$4|second|seconds}}```

@Nikerabbit If you could send an ok and I will add this ...

**Comment by Nikerabbit** [comment]
@kghbln Can I assume you figured out your question yourself, or is more clarification needed?

**Comment by kghbln** [comment]
@Nikerabbit I will take this as a yes. Thanks!
> (assuming that true is singular = from cache)

Was not to be checked by you in connection with this. :)

**Pull request merged**